### ISSUE TITLE
Virtualized collections have the wrong `aria-setsize`

### ISSUE THREAD

**Issue by NicholasBoll** [Workday/canvas-kit#2361]
## 🐛 Bug Report

Virtualized collections have the wrong `aria-setsize`

## To Reproduce

Go to https://codesandbox.io/s/summer-lake-zd68gl?file=/src/App.tsx and observe the `aria-setsize` of each item. It is `18` which is the pixel height of the item and not the total items in the set. It should be 1000

## Link to repl or repo (highly encouraged)

https://codesandbox.io/s/summer-lake-zd68gl?file=/src/App.tsx
